### DISCUSSION TITLE
Vercel error: Failed deployment

### DISCUSSION THREAD

**Discussion by aplasa85** [neuefische/questions#385] [Web] [answered]
Hello, 

I am still unable to make a pull request due to the latest mistake of vercel. My code is free of any mistake, all tests run good. Only remaining issue to be able to merge and move on. I cannot relate what the meaning of this mistake is. Does anybody have any clue?

<img width="1406" alt="image" src="https://user-images.githubusercontent.com/107868311/206458920-e7b38f62-3593-4b4b-a6ff-0b8f7cd9c73b.png">

Axel

**Comment by actyvystom** [comment, marked as answer]
It is telling you the error (in your _app.js file):
`Module not found: Can't resolve...`
Obviously you are trying to import a component which does not exist - at least at this path `../components/Layout`!

**Reply by aplasa85** [reply]
So far cleaned all the excess code and other points. Still if I change the path of Layout, then the page crashes. Otherwise, it does not pass the test of Vercel...Don't know what else should I put instead of the path of Layout.

**Reply by aplasa85** [reply]
It seems that the page is perfectly fine. Only the fact that the import of Layout fails the test of Vercel as it seems.
